frontend/talentsearch/src/js/components/signAndSubmit/SignAndSubmitPage.tsx

```diff
@@ -0,0 +1,284 @@
+
+  const tocNavItems = [
+    {
+      id: "importantInformation",
+      title: intl.formatMessage({
+        defaultMessage: "Important information",
+        description: "Toc navigation item on sign and submit page.",
+      }),
+      component: <ImportantInfo />,
+      icon: UserIcon,
+    },
+    {
+      id: "signature",
+      title: intl.formatMessage({
+        defaultMessage: "Signature",
+        description: "Toc navigation item on sign and submit page.",
+      }),
+      component: <Signature isNotComplete={isNotComplete} />,
+      icon: ClipboardCheckIcon,
+    },
+  ];
+
+  const applicationRoute = "#REPLACE-WITH-APPLICATION-ROUTE";
+  return (
+    <ApplicationPageWrapper
+      closingDate={closingDate}
+      title={intl.formatMessage({
+        defaultMessage: "My application profile",
+        description: "Title for sign and submit page.",
+      })}
+      crumbs={[
+        {
+          title: intl.formatMessage({
+            defaultMessage: "My applications",
+            description: "Breadcrumb for sign and submit page.",
+          }),
+          href: paths.allPools(),
+          icon: <BriefcaseIcon style={{ width: "1rem", marginRight: "5px" }} />,
+        },
+        {
+          title: jobTitle,
+          href: applicationRoute,
+        },
+        {
+          title: intl.formatMessage({
+            defaultMessage: "Step 2",
+            description: "Breadcrumb for sign and submit page.",
+          }),
+        },
+      ]}
+      navigation={{
+        currentStep: 2,
+        steps: [
+          {
+            path: applicationRoute,
+            label: intl.formatMessage({
+              defaultMessage: "Step 1: Review my profile",
+              description: "Navigation step in sign and submit page.",
+            }),
+          },
+          {
+            path: "#sign-and-submit",
+            label: intl.formatMessage({
+              defaultMessage: "Step 2: Sign and submit",
+              description: "Navigation step in sign and submit page.",
+            }),
+          },
+        ],
+      }}
+      subtitle={jobTitle}
+    >
+      <TableOfContents.Wrapper>
+        <TableOfContents.Navigation>
+          {tocNavItems.map((item) => (
+            <TableOfContents.AnchorLink key={item.id} id={item.id}>
+              {item.title}
+            </TableOfContents.AnchorLink>
+          ))}
+        </TableOfContents.Navigation>
+        <TableOfContents.Content>
+          {tocNavItems.map((item) => (
+            <TableOfContents.Section key={item.id} id={item.id}>
+              <TableOfContents.Heading as="h3" icon={item.icon}>
+                {item.title}
+              </TableOfContents.Heading>
+              {item.component}
+            </TableOfContents.Section>
+          ))}
+        </TableOfContents.Content>
+      </TableOfContents.Wrapper>
+    </ApplicationPageWrapper>
+  );
+};
+
+const SignAndSubmitPage: React.FC<{ id: string }> = ({ id }) => {
```

Comment:
`React.FC` is not longer reccomened and we are slowly moving to:

`const Component = ({props}: PropsInterface) => {}`

Reply:
Fixed by [Remove React.FC type from components](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3678/commits/c8632abf5980eac17565ab2f878ec1498f93d15e)